android/libraries/rib-debug-utils/src/main/java/com/uber/rib/core/RouterDebugUtils.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rib.core
+
+/** Debugging utilties when working with Routers.  */
+object RouterDebugUtils {
```

Comment:
If you do this in two commits under 1 pr, and don't squash, you'll keep the git history for the file move.

Reply:
good point, will do